tasks/managed/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       default: "3"
     - name: concurrentLimit
       type: string
-      default: 5
+      default: 90
```

Comment:
I think the jump from 5 to 90 too big. Also it might require increasing the pod limits as more cosigns would be hanging on the pod. Maybe we should try a smaller increase?

Reply:
I think not testing this for smaller snapshots is a pretty big red flag for merging this. The vast majority of users (at least AFAIK) are smaller snapshots. We are 18x'ing the concurrency limit for those and haven't even looked at what it does to them :upside_down_face: All in the favor of one large user